packages/bpk-animate-height/readme.md

```diff
@@ -43,10 +43,11 @@ class AnimateHeightContainer extends Component {
 
 ## Props
 
-| Property    | PropType         | Required | Default Value |
-| ----------- | ---------------- | -------- | ------------- |
-| children    | node             | true     | -             |
-| duration    | number           | true     | -             |
-| height      | string or number | true     | -             |
-| easing      | string           | false    | 'ease'        |
-| style       | object           | false    | {}            |
+| Property                 | PropType         | Required | Default Value |
+| -----------              | ---------------- | -------- | ------------- |
+| children                 | node             | true     | -             |
+| duration                 | number           | true     | -             |
+| height                   | string or number | true     | -             |
+| easing                   | string           | false    | 'ease'        |
```

Comment:
Do we have a list of accepted values for the `easing` prop somewhere? It's worth adding it to the docs/readme IMHO

Reply:
Are you thinking something like this?
https://github.com/Skyscanner/backpack/pull/229/files#diff-9e048b7a268bb81c6f466782a011779dR8

Reply:
Sorry I didn't see that bit, ignore me!

Reply:
That's because I only just added it!

Reply:
ah! In which case don't ignore me 😄 but yes perfect, that's what I was talking about